audio/CMakeLists.txt

```diff
@@ -31,7 +31,7 @@ include_directories (
       ${CMAKE_CURRENT_BINARY_DIR}
       ${CMAKE_CURRENT_LIST_DIR}
       ${PROJECT_SOURCE_DIR}/global
-      ${PROJECT_SOURCE_DIR}/thirdparty # because #include "mscore/musescore.h"
+      ${PROJECT_SOURCE_DIR}/thirdparty  #because #include "mscore/musescore.h"
```

Comment:
A `git commit --amend` save the commit message unchanged and `git push -f` would have do that too ;-)

Reply:
Thanks. 
I knew that there would be conflicts after the merger of another PR, so I just made it a separate commit :)

Reply:
And indeed a rebase is needed now ;-)

Reply:
Also it didn't help, Travis still failed